airbyte-webapp/src/components/ImageBlock/ImageBlock.tsx

```diff
@@ -9,18 +9,28 @@ interface ImageBlockProps {
   img?: string;
   num?: number;
   small?: boolean;
+  color?: string;
+  light?: boolean;
+  ariaLabel?: string;
 }
 
-export const ImageBlock: React.FC<ImageBlockProps> = ({ img, num, small }) => {
+export const ImageBlock: React.FC<ImageBlockProps> = ({ img, num, small, color, light, ariaLabel }) => {
   const imageCircleClassnames = classnames({
     [styles.circle]: num,
     [styles.iconContainer]: !num || num === undefined,
     [styles.small]: small && !num,
-    [styles.darkBlue]: !small && num,
+    [styles.darkBlue]: !small && num && !color,
+    [styles.green]: color === "green",
+    [styles.red]: color === "red",
+    [styles.blue]: color === "blue",
+    [styles.light]: light,
   });
+
+  const numberStyles = classnames(styles.number, { [styles.light]: light });
+
   return (
-    <div className={imageCircleClassnames}>
-      {num ? <div className={styles.number}>{num}</div> : <div className={styles.icon}>{getIcon(img)}</div>}
+    <div className={imageCircleClassnames} aria-label={ariaLabel}>
```

Comment:
Is `aria-label` right here?  I believe that's what you said, though I can't remember why that and not `aria-description`.  This isn't an interactive element.

Good news in general: We can now start adding `aria-labels` other places that use this component as well (this is used to render logos on our list pages, the little circles with the numbers in it on list pages, etc.)